content/docs/concepts/csidriver/features/powerflex.md

```diff
@@ -1081,3 +1081,46 @@ The Secret specifies the zone associated with each PowerFlex system along with a
         pools:
           - "pool2"
 ```
+## Expose the SFTP settings to automatically pull the scini.ko kernel module
+PowerFlex CSI driver version 2.14.0 and above supports exposing the SFTP settings to automatically pull the scini.ko kernel module.
+
+This feature allows the configuration of SFTP settings to automatically pull the scini.ko kernel module from a specified SFTP repository or SFTP repository of Dell. This ensures that the necessary kernel module is available and up-to-date on the cluster.
+
+The driver will try to find the kernel modules in the following order:
+
+- Pre-compiled Modules:
+
+Driver will fetch pre-compiled kernel modules for various Linux kernels that are typically included in PowerFlex SDC installation packages. These modules are stored in a driver cache. 
+
+- Automatic Fetching from SFTP repository:
+
+Driver will fetch the necessary kernel modules from a remote SFTP repository. This is configured through environment variables and secrets in the deployment templates.
+
+- Automatic SDC Compilation:
+
+If a matching pre-compiled module or the kernel module from SFTP repository is not available, driver will compile the SDC driver automatically. This involves compiling the scini.ko kernel module to match the running kernel.
+
+### Configuration Steps
+
+#### Enable SFTP Repository Settings:
+
+Enable the SFTP repository settings by enabling the SDC SFTP Repo and configuring the SFTP repository address, username, and secret.
+
+
+ *NOTE:*
+- Exposing SFTP settings to automatically pull scini.ko modules is only available for SDC 3.6.5 and 4.5.4
```

Comment:
do we need to mention version here ?

Reply:
yes Prasanna, the feature doesnt work in other sdc versions